doc/classes/CompositorEffect.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="CompositorEffect" inherits="Resource" is_experimental="true" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		This resource allows for creating a custom rendering effect.
+	</brief_description>
+	<description>
+		This resource defines a custom rendering effect that can be applied to [Viewport]s through the viewports [Environment]. You can implement a callback that is called during rendering at a given stage of the rendering pipeline and allows you to insert additional passes. Note that this callback happens on the rendering thread.
+		[b]Note:[/b] This functionality is still experimental, there will be changes to the implementation as we expose more of the rendering internals.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="_render_callback" qualifiers="virtual">
+			<return type="void" />
+			<param index="0" name="effect_callback_type" type="int" />
+			<param index="1" name="render_data" type="RenderData" />
+			<description>
+				Implement this function with your custom rendering code. [param effect_callback_type] should always match the effect callback type you've specified in [member effect_callback_type]. [param render_data] provides access to the rendering state, it is only valid during rendering and should not be stored.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="access_resolved_color" type="bool" setter="set_access_resolved_color" getter="get_access_resolved_color">
+			If [code]true[/code] and MSAA is enabled, this will trigger a color buffer resolve before the effect is run.
+			[b]Note:[/b] In _render_callback, to access the resolved buffer use:
```

Comment:
```suggestion
			[b]Note:[/b] In [code]_render_callback[/code], to access the resolved buffer use:
```
For readability, same for all these

Reply:
After changing that the commit checks nicely informed me I should be using `[member _render_callback]` :)